sim/priest/shadow/rotation.go

```diff
@@ -68,10 +69,15 @@ func (spriest *ShadowPriest) tryUseGCD(sim *core.Simulation) {
 	// initialize helpful variables for calculations later
 	vtCastTime := spriest.ApplyCastSpeed(time.Millisecond * 1500)
 	gcd := spriest.SpellGCD()
-	tickLength := spriest.ApplyCastSpeed(time.Second)
+	var mf_reduc_time time.Duration
+	if spriest.HasSetBonus(priest.ItemSetCrimsonAcolyte, 4) {
```

Comment:
It might be cleaner to have a helper function in mind_flay.go which returns the base channel tick time. Then you could use that here and also in the construction of the Mind Flay spell.

Reply:
I'll work on adding these optimizations later in code clean up, but I am trying to figure out core rotation stuff first and I suck at minor things like this.